OJS.Workers.SubmissionProcessors/Models/SubmissionInputModel.cs

```diff
@@ -15,6 +18,15 @@ public class SubmissionInputModel<TInput> : ISubmission
 
         public int TimeLimit { get; set; }
 
+        public string Code
+        {
+            get => this.code
+                   ?? (string.IsNullOrWhiteSpace(this.AllowedFileExtensions)
```

Comment:
This formatting seems odd. Either put `this.code` on new line or shift+tab the right-hand side of the expression.